client/src/transmitter/index.ts

```diff
@@ -42,47 +43,12 @@ console.log('pathname', pathname)
 
 const localhost = isLocalHost(hostname) || isIp(hostname)
```

Comment:
Is `hostname` a global here?

Reply:
yeah https://github.com/victorges/justcast.it/blob/83400725c544befd54a864dfbe0c0333b5d0cca9/client/src/transmitter/index.ts